### PR TITLE
qa: ignore PG health warnings in CephFS QA

### PR DESCRIPTION
**Pull request by batrick**
Fixes: https://tracker.ceph.com/issues/51282
Fixes: https://tracker.ceph.com/issues/65018

## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "quincy"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

- When filling out the below checklist, you may click boxes directly in the GitHub web UI.  When entering or editing the entire PR message in the GitHub web UI editor, you may also select a checklist item by adding an `x` between the brackets: `[x]`.  Spaces and capitalization matter when checking off items this way.

## Checklist
- Tracker (select at least one)
  - [x] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [ ] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [x] No impact that needs to be tracked
- Documentation (select at least one)
  - [ ] Updates relevant documentation
  - [x] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [x] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [ ] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
- `jenkins test rook e2e`
</details>

### CODE REVIEW

File: qa/cephfs/overrides/pg_health.yaml [hunk 1]
```diff
@@ -1,11 +1,13 @@
...
+# After discussions with Sam Just, here's why we always ignore these:
 #
-# https://tracker.ceph.com/issues/52624
+# These cluster health warnings are not concerning within the context of CephFS
+# QA testing. Generally, we would not usually see the warnings because OSD past
+# interval sets are not changed for the duration of the test. However,
+# sometimes during pool creation or when the upmap balancer is swapping OSDs
+# PGs can become degraded (no write I/O) which can be unavoidable during brief
+# recovery. (For the upmap balancer, log-based recovery instead of backfill can
+# trigger this.)
```
Comment (athanatos) [inline qa/cephfs/overrides/pg_health.yaml:9]: How about

```
# PG_DEGRADED warnings are not generally indicative of a bug.
# Any acting set changes (such as those caused by the upmap
# balancer to improve OSD space usage balance) can cause objects
# to become degraded transiently.
```

**Review by athanatos — APPROVED** [review]
Fine otherwise.

File: qa/cephfs/overrides/pg_health.yaml (hunk 1 above)
Comment (batrick) [reply qa/cephfs/overrides/pg_health.yaml]: done, thanks

**Comment by batrick** [comment]
jenkins test make check

**Comment by batrick** [comment]
jenkins test make check arm64

File: qa/cephfs/overrides/pg_health.yaml [hunk 2]
```diff
@@ -1,11 +1,9 @@
-# To prevent PG_AVAILABILITY failures from dominating runs, we usually ignore
-# these warnings. However, we'd like to see the cause of these warnings to be
-# fixed so occasionally allow the warning to fail the job.
+# After discussions with Sam Just, here's why we always ignore these:
```
Comment (athanatos) [inline qa/cephfs/overrides/pg_health.yaml:1]: My name doesn't need to be here.

Comment (batrick) [reply qa/cephfs/overrides/pg_health.yaml]: fixed

**Comment by batrick** [comment]
https://tracker.ceph.com/issues/65215#note-8

**Pull request merged**